SpatialGDKEditorCommandlet/Source/Private/Commandlets/SnapshotCommandlet.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SnapshotCommandlet.h"
+#include "SpatialGDKEditorCommandletPluginPrivate.h"
+#include "SpatialGDKEditor.h"
+#include "Runtime/Engine/Classes/Kismet/GameplayStatics.h"
+
+#include "Runtime/Engine/Classes/Engine/ObjectLibrary.h"
+#include "Runtime/Engine/Classes/Engine/World.h"
+#include "FileHelpers.h"
+
+USnapshotCommandlet::USnapshotCommandlet()
+{
+	IsClient = false;
```

Comment:
Could an initializer list be used here?

Reply:
No, initializer list is only good for members of the class (not members of the base class, which is what these are)